integration-tests/manifests/cmd/validate_instrumentation_vars.go

```diff
@@ -51,7 +61,7 @@ func main() {
 	}
 }
 
-func verifyInstrumentationEnvVariables(clientset *kubernetes.Clientset, namespace, jsonPath string) bool {
+func verifyInstrumentationEnvVariables(clientset *kubernetes.Clientset, namespace, jsonPath string, appSignals string) bool {
```

Comment:
Instead of passing an argument can we look at the agent config instead for checking if app signals is enabled?